packages/utils/src/lib/__snapshots__/report-to-md.spec.ts.snap

```diff
@@ -1,533 +1,440 @@
 </details>
 
-- <details>
-<summary>Require braces around arrow function bodies (1) ESLint</summary>
-1 warning
-ESLint rule **arrow-body-style**.
-[Documentation](https://eslint.org/docs/latest/rules/arrow-body-style)
 
-</details>
+ESLint rule **no-unused-vars**. [📖 Docs](https://eslint.org/docs/latest/rules/no-unused-vars)
 
-- <details>
-<summary>Enforce camelcase naming convention (1) ESLint</summary>
-undefined
-ESLint rule **camelcase**.
-[Documentation](https://eslint.org/docs/latest/rules/camelcase)
+### Require calls to \`isNaN()\` when checking for \`NaN\` (ESLint)
 
-</details>
+🟩 <b>0</b> (score: 100)
 
-- <details>
-<summary>Enforce consistent brace style for all control statements (1) ESLint</summary>
-undefined
-ESLint rule **curly**.
-[Documentation](https://eslint.org/docs/latest/rules/curly)
+ESLint rule **use-isnan**. [📖 Docs](https://eslint.org/docs/latest/rules/use-isnan)
 
-</details>
+### Enforce comparing \`typeof\` expressions against valid strings (ESLint)
 
-- <details>
-<summary>Require the use of \`===\` and \`!==\` (1) ESLint</summary>
-1 warning
-ESLint rule **eqeqeq**.
-[Documentation](https://eslint.org/docs/latest/rules/eqeqeq)
+🟩 <b>0</b> (score: 100)
 
-</details>
+ESLint rule **valid-typeof**. [📖 Docs](https://eslint.org/docs/latest/rules/valid-typeof)
 
-- <details>
-<summary>Enforce a maximum number of lines of code in a function (1) ESLint</summary>
-1 warning
-ESLint rule **max-lines-per-function**.
-[Documentation](https://eslint.org/docs/latest/rules/max-lines-per-function)
+### Require braces around arrow function bodies (ESLint)
 
+<details>
+<summary>🟥 <b>1 warning</b> (score: 0)</summary>
+<h4>Issues</h4><table><tr><th>Severity</th><th>Message</th><th>Source file</th><th>Line(s)</th></tr><tr><td>⚠️ <i>warning</i></td><td>Unexpected block statement surrounding arrow body; move the returned value immediately after the \`=>\`.</td><td><code>src/components/TodoFilter.jsx</code></td><td>3-25</td></tr></table>
 </details>
 
-- <details>
-<summary>Enforce a maximum number of lines per file (1) ESLint</summary>
-undefined
-ESLint rule **max-lines**.
-[Documentation](https://eslint.org/docs/latest/rules/max-lines)
 
-</details>
+ESLint rule **arrow-body-style**. [📖 Docs](https://eslint.org/docs/latest/rules/arrow-body-style)
 
-- <details>
-<summary>Require or disallow method and property shorthand syntax for object literals (1) ESLint</summary>
-3 warnings
-ESLint rule **object-shorthand**.
-[Documentation](https://eslint.org/docs/latest/rules/object-shorthand)
+### Enforce camelcase naming convention (ESLint)
 
-</details>
+🟩 <b>0</b> (score: 100)
+
+ESLint rule **camelcase**. [📖 Docs](https://eslint.org/docs/latest/rules/camelcase)
+
+### Enforce consistent brace style for all control statements (ESLint)
+
+🟩 <b>0</b> (score: 100)
+
+ESLint rule **curly**. [📖 Docs](https://eslint.org/docs/latest/rules/curly)
```

Comment:
Hmm, in category detail we now have description 1st and result (score) 2nd, while in audit detail we have result (value+score) 1st and description 2nd. I guess it would be better for them to be consistent :thinking:

Maybe always put description first? What do you think?

Reply:
It's a good point to discuss. While I'd keep everything consistent (not to name it "same"), I would also make sure it works from the UX perspective. For me personally, the comparison of "categories" and "audits" section cannot be equal:
- categories section should provide basic info - you see the list of multiple audits, and when colour marker is 1st, you're able to determine the audit which interests you more;
- audits section provides explicit info - usually you either autoscrolled here from categories, or find the audit yourself, you look for the specific one, and its name on 1st position makes sense.
I personally like how it works now, it's easy to navigate and focus on the most important data. @matejchalk if you think there's a better approach, let's keep it either till Monday daily or the refinement meeting. I would also be happy to know the opinion of @Tlacenka on the UX here.

Reply:
From my POV it depends on whether the category description is supposed to be important (and how often you would want to read it).

Initially, I thought that we could reuse the same collapsible strategy from the audit but in categories, you wouldn't want them collapsed by default. For that reason, if the description is actually informative, it should be at the top.

As for the audits, since they have collapsible details and the main point is their score, I would keep description and link at the bottom.

While this introduces inconsistencies, I agree with @MishaSeredenkoPushBased that each type of entity has a different context and therefore this inconsistency is fine IMO. Should we want to enforce the inconsistency no matter what, we can put the audit description at the top.

Reply:
Yeah, I don't have strong feelings on this one, so I'm fine with keeping it as is.

If we were to change it, I would put the description on top for audits as well, not the other way around, because for categories the description would be hard to notice underneath the list.